data_preparation/prepare_data_full_games.ipynb

```diff
@@ -34,13 +34,13 @@
     "import time\n",
     "start = time.time()\n",
     "\n",
-    "df = pd.read_csv('chess_games.csv',nrows=1_000)\n",
+    "df = pd.read_csv('D:\\Programowanie\\Projekty\\chess_cheater_project\\datasets\\chess_games.csv',nrows=1_000)\n",
```

Comment:
może nie hardcodować tego tylko jakiegoś os.patha czy coś ? analogicznie do wczesniejszego requesta. Jak nie masz czasu to ja się tym mogę tez zająć żeby to przerobic na jakieś pośrednie i spójne z ułożeniem folderów polecenia (nie robiłem tego a się przyda)